traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/StatTracker.java

```diff
@@ -184,7 +184,13 @@ public void setRegionalAlternateCount(final int regionalAlternateCount) {
 			/**
 			 * This value indicates that routing was blocked in accordance with anonymous blocking configurations.
 			 */
-			ANON_BLOCK
+			ANON_BLOCK,
+			/**
+			 * This value indicates that routing was based on the default lat/long of the delivery service, because maxmind
+			 * returned the centre of the country as the client location, due to the CZF not being able to resolve the client IP
+			 * to a valid location.
+			 */
+			GEO_DS
```

Comment:
do you think there would be any gain to add another result type that let us know that it is just using the default from maxmind (like if it doesnt have a DS default and the client ip didnt resolve to a valid location)? not sure if theres any gain there but might just give us more info quickly?

Reply:
This was part of the ticket description from @dneuman64. I think he wants it so that the Maple team can easily see how many events matched this type.

Reply:
yea thats why i was thinking it might be good to add another one in there that tells us that it is just using the maxmind defaults. eh if theres value there itll be an easy change to add in if people decide they want it